go/signedexchange/cbor/types.go

```diff
@@ -0,0 +1,34 @@
+// Package cbor defines a parser and encoder for a subset of CBOR, RFC7049.
+//
+// Supported:
+//  * Major types:
+//     * 0: Unsigned integers, with both minimal and fixed-length 64-bit
+//          (additional information 27) encodings.
+//     * 1: Negative integers, with minimal encoding for integers.
+//     * 2 & 3: Byte and UTF-8 strings, with minimal encoding for lengths.
+//     * 4 & 5: Arrays and maps, with the number of elements known at the start
+//              of the container, encoded minimally.
+//  * Pre-encoded items, by copying from a Reader.
+//  * Retrieval of the current byte offset within an array or map.
+//  * Items that don't fit in memory.
+//
+// Unsupported:
+//  * Negative integers (major type 1) between -2^63-1 and -2^64 inclusive,
+//    since they don't fit in a 2's-complement int64.
+//  * Floating-point numbers
+//  * Indefinite-length encodings.
+//  * Parsing
+package cbor
+
+type Type byte
+
+const (
+	TypePosInt Type = iota << 5
```

Comment:
This enum values would be 0, 32, 64, 96, ... that exceeds `byte` values. Is this expected?

Reply:
Yeah, probably its better to be explicit here.

Reply:
Ah, as maximum values are 0xc0, this is not a problem. Please ignore the above comment.